data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3437 @@
+				decline
+
+
+
+outfit "Mystery Cube"
+	category "Special"
+	cost 1000000
+	thumbnail "outfit/unknown"
+
+ship "Marauder Leviathan" "Leviathan (Cube)"
+	outfits
+		"A370 Atomic Thruster"
+		"A525 Atomic Steering"
+		"Anti-Missile Turret"
+		"D94-YV Shield Generator"
+		"Fusion Reactor"
+		Hyperdrive
+		"LP144a Battery Pack"
+		"Laser Rifle" 69
+		"Security Station" 5
+		"Small Bunk Room"
+		"Liquid Helium Cooler"
+		"Luxury Accommodations"
+		"Proton Turret" 3
+		Torpedo 102
+		"Torpedo Launcher" 4
+		"Torpedo Storage Rack" 6
+	cargo
+		outfits
+			"Gold Bars" 15
+			"Mystery Cube"
+
+mission "Unfettered: Special Cargo"
+	name `Mystery item`
+	description `Go to <waypoints> and find a way to acquire the 'unique' item from the prominent warlord there. His ship is the Degenerate. Then bring the item to He Yi, on Warfeed.`
+	to offer
+		has "Unfettered: Solifuge 9: Shield Beetles Mass Production: done"
+		has "reputation: Hai" >= 0
+	source "Darkcloak"
+	waypoint "Alcyone"
+	destination "Warfeed"
+	on offer
+		conversation
+			`You are starting to get quite used to Unfettered spaceports, and strangely you are not even surprised by the amount of damage there is around on different non essentials structures anymore. At least the shipyards, hospitals and the stadium of the council are all in perfect shape, that is what matters most after all.`
+			`	You want to see a bit of change for once, and wander on a side of the spaceport you have not explored before, but it does not seem that much different than the rest.`
+			`	You get interrupted in your thinking by a local Unfettered, he seems to have a very important mission to give you, given how fast he is speaking. "He Yi has asked me to deliver you a message! It has come to his attention that an item of great importance to us is currently in the possession of an unfettered human warlord." You guess he is referring to a pirate leader. "Said warlord can be found in <waypoints>, we want you to go and retrieve that item right now, in whatever way you deem fit."`
+			choice
+				`	"How will I know which warlord it is?"`
+				`	"This seems too risky for me."`
+					decline
+			`	"It is no common warlord. To say the truth it is the most powerful warlord there, and his ship is the 'Degenerate', you may have to bring it down in order to complete the mission. He may accept a payment instead, but do remember, we are not willing to pay a fortune for this item so if you cannot face him you may have to debourse quite a large sum of your own money. The warlord, Mari, got the item from another warlord. Also you need not worry about how to find said item, it is... unique, you will understand once you see it. Once you have it, bring it back to my master, you have been given access to his war room, on <planet>." Mari sounds like the name of a woman to you, it is impressive she has managed to rise so high in power, among a pirate culture that is. But maybe you have misjudged them.`
```

Comment:
Well the pirate culture as I understand it is the strongest leads; and in those circumstances I thought that they would still have the backwards mentalities some still have today.
There is a female UHai warlord coming; and that isn't unlikely or anything, but for pirates I thought so; could be changed though. In FW one of the negotiators can't go because of his sexual orientation.